### PR TITLE
[RG-231] Status icon align to center

### PR DESCRIPTION
**Pull request by KochynVolodymyr**
### Motivate of the pull request
 - [x] To address an existing issue. If so, please provide a link to the issue: RG-231
 - [ ] Breaking new feature. If so, please describe details in the description part.

 ### Describe the technical details
Align status icon to the center:
image

 ### Which part of the code base require a change
 
 - [x] Library: compiler
 - [ ] Plug-in: <Specify the plugin name>
 - [ ] Engine
 - [ ] Documentation
 - [ ] Regression tests
 - [ ] Continous Integration (CI) scripts

### CODE REVIEW

File: src/Compiler/TaskTableView.cpp [hunk 1]
```diff
@@ -263,13 +263,26 @@ void TaskTableView::TasksDelegate::paint(QPainter *painter,
...
+      label->move(m_view.visualRect(index).center() -
+                  (label->rect().center() - QPoint{6, 0}));
       m_inProgressMovie->start();
       return;
     } else {
       // Reset the movie when task is no longer in progress
       label->setMovie(nullptr);
     }
+
+    QStyleOptionViewItem opt = option;
```
Comment (ravic-rs) [inline src/Compiler/TaskTableView.cpp:275]: What is this code trying to do ?

Comment (KochynVolodymyr) [reply src/Compiler/TaskTableView.cpp:275]: Create new object `opt` and copy all data from object `option`. Copy constructor is used here.

Comment (ravic-rs) [reply src/Compiler/TaskTableView.cpp:275]: For alignment, I see the code above but didn't follow why we need this change.

Comment (KochynVolodymyr) [reply src/Compiler/TaskTableView.cpp:275]: This code need for alignment. What exactly is not clear?

**Pull request merged**